packages/api/src/api/helix/chat/HelixEmote.ts

```diff
@@ -28,4 +55,47 @@ export class HelixEmote extends DataObject<HelixEmoteData> {
 	getImageUrl(scale: HelixEmoteImageScale): string {
 		return this[rawDataSymbol].images[`url_${scale}x` as const];
 	}
+
+	/**
+	 * Gets the URL of the emote image in static format at the given scale and theme mode, or null if a static emote image at that scale/theme mode doesn't exist.
+	 *
+	 * @param scale The scale of the image.
+	 * @param themeMode The theme mode of the image, either `light` or `dark`.
+	 */
+	getStaticImageUrl(scale: HelixEmoteScale = '1.0', themeMode: HelixEmoteThemeMode = 'light'): string | null {
+		if (this[rawDataSymbol].format.includes('static') && this[rawDataSymbol].scale.includes(scale)) {
+			this.getFormattedImageUrl(scale, 'static', themeMode);
```

Comment:
This doesn't return anything, and resulting from that, the method always returns `null`. (same for the other one)

Reply:
Fixed.